src/index.js

```diff
@@ -1,8 +1,23 @@
-const {ApolloServer, gql} = require('apollo-server')
+const { ApolloServer, gql } = require('apollo-server');
+const { Sequelize } = require('sequelize');
+require('dotenv').config();
+let postgresUrl = process.env.POSTGRES_URL;
```

Comment:
bit nitpicky but i dont see a reason to store in a variable if we're only using it once on line 5